lib/active_merchant/billing/gateways/secure_trading.rb

```diff
@@ -0,0 +1,274 @@
+require 'nokogiri'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    # For more information on the Secure Trading visit
+    # {SecureTraining}[https://docs.securetrading.com]
+    #
+    class SecureTradingGateway < Gateway
+      self.live_url = 'https://webservices.securetrading.net:443/xml/'
```

Comment:
I don't believe there's a need for add `:443` when specifying `https` because that's the default port for that scheme but ... doesn't change anything.